### DISCUSSION TITLE
Allow display of "all" comments - not just pages of them

### DISCUSSION THREAD

**Discussion by Papoulka** [mattwright324/youtube-comment-suite#119] [Ideas]
This is a great tool of course.

As an improvement:  In the results for a searched comment, let me choose "Show all" instead of pages of 500 at a time.  I realize that an Export will get me most of the same info but this would be a useful feature.

Thank you

**Comment by mattwright324** [comment]
The main concern with displaying all comments is that you could potentially have hundreds of thousands to several millions of comments and this would very likely freeze the application if not extremely slow and negatively impact user experience.

**Comment by Papoulka** [comment]
I understand, and can't think of a simple way around that.  Thank you.
